plugin/forward/metrics.go

```diff
@@ -1,14 +1,38 @@
 package forward
 
 import (
+	"strconv"
+	"time"
+
 	"github.com/coredns/coredns/plugin"
 
+	"github.com/miekg/dns"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promauto"
 )
 
 // Variables declared for monitoring.
 var (
+	RequestCount = promauto.NewCounterVec(prometheus.CounterOpts{
```

Comment:
Please make all of these metric vars unexported so they can't be used outside of the package.

```suggestion
	requestCount = promauto.NewCounterVec(prometheus.CounterOpts{
```

Reply:
Have unexported all metrics defined within the `forward` plugin.